tests/test_subscriptions.py

```diff
@@ -75,6 +77,7 @@ def setUp(self):
         self.sample_percolate_query = smartseq2_paired_ends_vx_query
         self.hmac_key_id = 'dss_test'
         self.hmac_secret_key = '23/33'
+        self.replicas = ['aws', 'gcp']
```

Comment:
Is there a programmatic way to get all replica names from a Replica object?

Reply:
Yes:
```
self.replicas = [replica.name for replica in Replica]
```

Reply:
`self.replicas = [replica.name for replica in Replica]` 🤔

Reply:
There is.  Will add it.  Thanks for catching that.